docs/API/Instance Methods.md

```diff
@@ -99,6 +99,48 @@ If an animation is started on a [keypath](../Concepts/Templates/Keypaths.md) whi
 
 ---
 
+## ractive.attachChild()
+
+Creates a parent-child relationship between two Ractive instances. The child may be an instance of a component defined by [extend](#extend), but that is not a requirement, as children may be a plain Ractive instance created with `new Ractive()`.
+
+**Syntax**
+```js
+ractive.attachChild( child );
+ractive.attachChild( child, options );
+```
+
+**Arguments**
+
+- `child (Ractive instance)`: The child instance to attach.
+- `options (Object)`: Options to use when attaching the child:
+> > * **target** *`string`* - an anchor name at which to render the instance. See [`Components`](../Extend/Components.md). If the instance is already rendered, it will be unrendered and re-rendered at an appropriate anchor.
```

Comment:
We should probably add an anchors section to the component docs and link directly to it.